features/unattached_commands.feature

```diff
@@ -196,6 +196,23 @@ Feature: Command behaviour when unattached
            | groovy   | no           |
            | hirsute  | no           |
 
+
+    @series.focal
```

Comment:
Can we have that test for all releases ?

Reply:
Yep good catch! I had just put one for getting it working, but will change this to "@series.all"